From 3b9dc3966fbfc2070b76353c2d24059b0adfe00c Mon Sep 17 00:00:00 2001
From: Sam Mortimer <sam@mortimer.me.uk>
Date: Thu, 9 Aug 2018 12:00:02 -0700
Subject: [PATCH 09/13] lineage-sdk: Use java.utils.Objects instead of
 libcore.util.Objects

Most methods in libcore.util.Objects have been deprecated:
https://android.googlesource.com/platform/libcore/+/44b7f98da7985691d592f6a6a9199593a9f6455d

Change-Id: I91bdd0c9756da153f167c80fe6d85fea7717a5da
---
 .../lineageos/platform/internal/display/TwilightTracker.java | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/lineage/lib/main/java/org/lineageos/platform/internal/display/TwilightTracker.java b/lineage/lib/main/java/org/lineageos/platform/internal/display/TwilightTracker.java
index e64f28d..eea25dc 100644
--- a/lineage/lib/main/java/org/lineageos/platform/internal/display/TwilightTracker.java
+++ b/lineage/lib/main/java/org/lineageos/platform/internal/display/TwilightTracker.java
@@ -38,8 +38,7 @@ import java.text.DateFormat;
 import java.util.ArrayList;
 import java.util.Date;
 import java.util.Iterator;
-
-import libcore.util.Objects;
+import java.util.Objects;
 
 /**
  * Figures out whether it's twilight time based on the user's location.
@@ -109,7 +108,7 @@ public final class TwilightTracker {
 
     private void setTwilightState(TwilightState state) {
         synchronized (mLock) {
-            if (!Objects.equal(mTwilightState, state)) {
+            if (!Objects.equals(mTwilightState, state)) {
                 if (DEBUG) {
                     Slog.d(TAG, "Twilight state changed: " + state);
                 }
-- 
2.17.1

